UI/DamageIndicator.cs

```diff
@@ -0,0 +1,42 @@
+using Godot;
+
+
+public class DamageIndicator : TextureRect
+{
+    Vector3 ShotFirePosition;
```

Comment:
This line has space based indentation (I can tell because Github displays tabs an 8 spaces), can you make sure that the whole file has tab indentation?